format/src/main/thrift/carbondata.thrift

```diff
@@ -183,11 +183,23 @@ struct FileFooter{
 }
 
 /**
+* Footer for indexed carbon file
```

Comment:
Can you change all comment for V3 format, mentioning this struct is for V3 format